File: Scripts/TermScripts/FunctionWords.pl
#!/usr/bin/perl -w
sub isFunctionWord {
    $term = shift;
    return ($term eq "e" || $term eq "em" || $term eq "uma" || $term eq "uma" || $term eq "la" || 
       $term eq "your" || $term eq "if" || $term eq "me" || $term eq "de" || $term eq "no" ||
       $term eq "por" || $term eq "of" || $term eq "about" || $term eq "either" || $term eq "by" ||
       $term eq "da" || $term eq "na" || $term eq "in" || $term eq "era" || $term eq "you" ||
       $term eq "sua" || $term eq "do" || $term eq "nos" || $term eq "ao" || $term eq "nao" ||
       $term eq "be" || $term eq "suas" || $term eq "dos" || $term eq "nas" || $term eq "sobre" ||
       $term eq "ou" || $term eq "our" || $term eq "onde" || $term eq "das" || $term eq "para" ||
       $term eq "the" || $term eq "onde" || $term eq "ser" || $term eq "quem" || $term eq "a" ||
       $term eq "com" || $term eq "se" || $term eq "entre" || $term eq "this" || $term eq "qual" ||
       $term eq "o" || $term eq "que" || $term eq "mais" || $term eq "sem" || $term eq "not" ||
       $term eq "my" || $term eq "as" || $term eq "como" || $term eq "sao" || $term eq "for" ||
       $term eq "are" || $term eq "meu" || $term eq "os" || $term eq "um" || $term eq "on" ||
       $term eq "to" || $term eq "here" || $term eq "aos");
}

1;

File: Scripts/TermScripts/FunctionWords.R

isFunctionWord <- function(term){
  if(term == "e" || term == "em" || term == "uma" || term == "la" || 
       term == "your" || term == "if" || term == "me" || term == "de" || term == "no" ||
       term == "por" || term == "of" || term == "about" || term == "either" || term == "by" ||
       term == "da" || term == "na" || term == "in" || term == "era" || term == "you" ||
       term == "sua" || term == "do" || term == "nos" || term == "ao" || term == "nao" ||
       term == "be" || term == "suas" || term == "dos" || term == "nas" || term == "sobre" ||
       term == "ou" || term == "our" || term == "onde" || term == "das" || term == "para" ||
       term == "the" || term == "ser" || term == "quem" || term == "a" ||
       term == "com" || term == "se" || term == "entre" || term == "this" || term == "qual" ||
       term == "o" || term == "que" || term == "mais" || term == "sem" || term == "not" ||
       term == "my" || term == "as" || term == "como" || term == "sao" || term == "for" ||
       term == "are" || term == "meu" || term == "os" || term == "um" || term == "on" ||
       term == "to" || term == "here" || term == "aos"){
    return(1)
  } else{
    return(0)
  }
  
}